pulsar-client-tools/src/main/java/org/apache/pulsar/admin/cli/CmdSinks.java

```diff
@@ -403,6 +412,35 @@ void processArguments() throws Exception {
                 sinkConfig.setResources(resources);
             }
 
+            // window configs
+            WindowConfig windowConfig = sinkConfig.getWindowConfig();
+            if (null != windowLengthCount) {
+                if (windowConfig == null) {
+                    windowConfig = new WindowConfig();
+                }
+                windowConfig.setWindowLengthCount(windowLengthCount);
+            }
+            if (null != windowLengthDurationMs) {
+                if (windowConfig == null) {
+                    windowConfig = new WindowConfig();
+                }
+                windowConfig.setWindowLengthDurationMs(windowLengthDurationMs);
+            }
+            if (null != slidingIntervalCount) {
```

Comment:
I don't think we should have sliding windows for batched sink.  What would be the use case for that?

Reply:
trying to make one up. think about a influxdb type sink where you want to write the average of some value over the last few seconds on a sliding basis?

Reply:
We are introducing a batched sink not a windowed sink thus I don't think the batched sink should have the same semantics and configs as a windowed function.  This will be very confusing to users.  To start with we should just start with a batchSize or batchTime configs and be distinct from windowing configs.